tests/main/try/task.yaml

```diff
@@ -0,0 +1,65 @@
+summary: Check that try command works
+
+environment:
+  PORT: 8081
+  SERVICE_FILE: "./service.sh"
+  SERVICE_NAME: "test-service"
+
+prepare: |
+  echo "Given a service listening on a port"
+  echo "#!/bin/sh\nwhile true; do echo \"HTTP/1.1 200 OK\n\nok\n\" |  nc -l -p $PORT -q 1; done" > $SERVICE_FILE
+  chmod a+x $SERVICE_FILE
+  systemd-run --unit $SERVICE_NAME $SERVICE_FILE
+  while ! netstat -lnt | grep -Pq "tcp.*?:$PORT +.*?LISTEN\n*"; do sleep 0.5; done
+
+restore: |
+  systemctl stop $SERVICE_NAME
+  rm -f $SERVICE_FILE
+
+execute: |
+  echo "Given a buildable snap in a known directory"
+  echo "When try is executed on that directory"
+  snap try ../../lib/snaps/basic-binaries
```

Comment:
We've been using `$TESTSLIB` instead of `../../lib` elsewhere. This will remain working even if we move the test around.